bin/lint-readme

```diff
@@ -68,4 +68,11 @@ node_modules/.bin/remark \
 cp README.md README.md.temp
 rewrite <README.md.temp >README.md
 
-node_modules/.bin/eslint -- README.md
+module="$(get module-type)"
```

Comment:
Let's move this alongside the other uses of `get` to make it easy to see which variables this script uses.

Reply:
:zap: